pages/index.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import Layout from '../components/Layout';
 import EventList from '../components/EventList';
-import EVENTS from '../fixtures/events'
+import { fixtureApi } from '../services';
```

Comment:
OK. I think I was trying to set up a config variable you could use to turn fixtures off and on. If you think that is a good idea maybe have that happen in index.